RadixWallet/Profile/Entity/Persona.swift

```diff
@@ -166,3 +166,15 @@ extension Profile.Network.Personas {
 		filter(\.isHidden).asIdentifiable()
 	}
 }
+
+extension Profile.Network.Persona {
+	public var shouldWriteDownSeedPhrase: Bool {
```

Comment:
This cannot be merged as is. SHOULD also check: `secureStorageClient.containsMnemonic(deviceFactorSource.id)`, since it is possible for that to be `false` (missing in keychain!) and for `backedUpIds` to not contains `deviceFactorSource.id`, in which case this current impl would return `true` for `shouldWriteDownSeedPhrase`, but the mnemonic is missing!